ui-frontend/packages/catalog-ui-search/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "catalog-ui-search",
   "author": "Codice",
   "description": "A frontend UI for DDF Catalog",
-  "version": "0.0.1",
+  "version": "3.0.0",
```

Comment:
should this be the next released version?

Reply:
I didn't know so I just checked a popular project (Material UI) and they have it set how I do - because there is no such thing as a snapshot version in js, the current version matches what's in master.

![image](https://user-images.githubusercontent.com/7236299/73859152-d4722a00-47f6-11ea-967c-1fc684c60dfc.png)
 https://material-ui.com/ and https://github.com/mui-org/material-ui/blob/master/package.json#L2